src/Features/Core/Portable/Workspace/MiscellaneousFileUtilities.cs

```diff
@@ -47,13 +47,14 @@ internal static ProjectInfo CreateMiscellaneousProjectInfoForDocument(
 
         if (parseOptions != null &&
             compilationOptions != null &&
+            languageInformation.ScriptExtension != "" &&
```

Comment:
Alternate option would be making ScriptExtension nullable.

Reply:
I think nullable would be more clear here (if not too difficult).

Reply:
Yeah, that was really easy. I should have thought of it first :)